WalletWasabi.Fluent/Helpers/KeyManagerExtensions.cs

```diff
@@ -1,17 +1,28 @@
 using System.Collections.Generic;
-using System.Linq;
-using NBitcoin;
 using WalletWasabi.Blockchain.Analysis.Clustering;
 using WalletWasabi.Blockchain.Keys;
-using WalletWasabi.Extensions;
 
 namespace WalletWasabi.Fluent.Helpers;
 
 public static class KeyManagerExtensions
 {
-	public static IEnumerable<LabelsArray> GetChangeLabels(this KeyManager km) =>
-		km.GetKeys(isInternal: true).Select(x => x.Labels);
+	public static (List<LabelsArray>, List<LabelsArray>) GetLabels(this KeyManager km)
+	{
+		var changeLabels = new List<LabelsArray>();
+		var receiveLabels = new List<LabelsArray>();
 
-	public static IEnumerable<LabelsArray> GetReceiveLabels(this KeyManager km) =>
-		km.GetKeys(isInternal: false).Select(x => x.Labels);
+		foreach (var key in km.GetKeys())
+		{
+			if (key.IsInternal)
+			{
+				changeLabels.Add(key.Labels);
```

Comment:
Instead of using `SelectMany` to mapconcat, i would suggest to have:

```c#
var changeLabels = new List<string>();
changeLabels.AddRange(Key.Labels);
```

Also, this operation is really common and useful in many scenarios. In F# it is called `partition` :

> Splits the collection into two collections, containing the elements for which the given predicate returns True and False respectively. Element order is preserved in both of the created lists.

I would create the extension method as:
```c#
public static class IEnumerableExtensions
{
   public static (IEnumerable<T>, IEnumerable<T>) Partition<T>(this IEnumerable<T> me, Predicate<T> predicate) {....
}
```